operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -2487,6 +2487,37 @@ nginx:
         }
       }
 
+# Use either this ingress or the gateway, but not both at once.
+# If you enable this, make sure to disable the gateway.
+ingress:
+  enabled: false
+#  ingressClassName: nginx
+  annotations: {}
+  paths:
+    distributor-headless:
+      - /distributor
```

Comment:
i see that the other ingresses have `pathType` configurable under the pretext that some ingress controllers require it ([code](https://github.com/grafana/mimir/blob/fa26805cebb1cc7dd0af21eb397aa3120f272152/operations/helm/charts/mimir-distributed/values.yaml#L2624-L2625)). Should we also support that? Something like

```yaml
distributor-headless:
  - path: /distributor
    # pathType: Prefix
```